### PR TITLE
Added steps for hybrid scenario

### PR DESCRIPTION
**Pull request by Batre-MSFT**
Added steps for hybrid scenario

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/collaboration-exo/public-folders/assign-permissions-mail-enabled-pfs.md [hunk 1]
```diff
@@ -55,3 +55,22 @@ For detailed syntax and parameter information, see the following articles:
 - [Set-MailPublicFolder](https://docs.microsoft.com/powershell/module/exchange/set-mailpublicfolder)
 
 - [Add-RecipientPermission](https://docs.microsoft.com/powershell/module/exchange/add-recipientpermission)
+
+## Send As mail enabled public folder in Hybrid scenario
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/collaboration-exo/public-folders/assign-permissions-mail-enabled-pfs.md:59]:
```suggestion
## Send As mail-enabled public folder in a hybrid scenario
```

File: Exchange/ExchangeOnline/collaboration-exo/public-folders/assign-permissions-mail-enabled-pfs.md [hunk 2]
```diff
@@ -55,3 +55,22 @@ For detailed syntax and parameter information, see the following articles:
...
+
+For Exchange Online mailboxes accessing public folders deployed at On-Premises:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/collaboration-exo/public-folders/assign-permissions-mail-enabled-pfs.md:61]:
```suggestion
For Exchange Online mailboxes accessing public folders deployed on-premises:
```

File: Exchange/ExchangeOnline/collaboration-exo/public-folders/assign-permissions-mail-enabled-pfs.md [hunk 3]
```diff
@@ -55,3 +55,22 @@ For detailed syntax and parameter information, see the following articles:
...
+
+1) Ensure Mail Enabled Public Folders are synced to Exchange Online
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/collaboration-exo/public-folders/assign-permissions-mail-enabled-pfs.md:63]:
```suggestion
1) Ensure that mail-enabled public folders are synced to Exchange Online.
```

File: Exchange/ExchangeOnline/collaboration-exo/public-folders/assign-permissions-mail-enabled-pfs.md [hunk 4]
```diff
@@ -55,3 +55,22 @@ For detailed syntax and parameter information, see the following articles:
 - [Set-MailPublicFolder](https://docs.microsoft.com/powershell/module/exchange/set-mailpublicfolder)
 
 - [Add-RecipientPermission](https://docs.microsoft.com/powershell/module/exchange/add-recipientpermission)
+
+## Send As mail enabled public folder in Hybrid scenario
+
+For Exchange Online mailboxes accessing public folders deployed at On-Premises:
+
+1) Ensure Mail Enabled Public Folders are synced to Exchange Online 
+Use following EXO PowerShell command to ensure On-Premises mail public folder are synced:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/collaboration-exo/public-folders/assign-permissions-mail-enabled-pfs.md:64]:
```suggestion
Use the following EXO PowerShell command to ensure that on-premises mail public folders are synced:
```

File: Exchange/ExchangeOnline/collaboration-exo/public-folders/assign-permissions-mail-enabled-pfs.md [hunk 5]
```diff
@@ -55,3 +55,22 @@ For detailed syntax and parameter information, see the following articles:
...
+
+`Get-MailPublicFolder <MEPFName>`
+Example:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/collaboration-exo/public-folders/assign-permissions-mail-enabled-pfs.md:67]:
```suggestion
Example
```

File: Exchange/ExchangeOnline/collaboration-exo/public-folders/assign-permissions-mail-enabled-pfs.md [hunk 6]
```diff
@@ -55,3 +55,22 @@ For detailed syntax and parameter information, see the following articles:
...
+Following example lists MEPF named OnPremPF
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/collaboration-exo/public-folders/assign-permissions-mail-enabled-pfs.md:68]:
```suggestion
The following example lists mail-enabled public folders named OnPremPF:
```

File: Exchange/ExchangeOnline/collaboration-exo/public-folders/assign-permissions-mail-enabled-pfs.md [hunk 7]
```diff
@@ -55,3 +55,22 @@ For detailed syntax and parameter information, see the following articles:
...
+
+`Get-MailPublicFolder OnPremPF`
+
+If the MEPF from On-Premises are not showing in EXO, use the Sync-MailPublicFolders.ps1 (for Exchange Server 2010) or Sync-ModernMailPublicFolders.ps1 (For Exchange 2013/2016/2019) to sync the MEPF's first.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/collaboration-exo/public-folders/assign-permissions-mail-enabled-pfs.md:72]:
```suggestion
If the on-premises mail-enabled public folders are not displayed, use Sync-MailPublicFolders.ps1 (for Exchange Server 2010) or Sync-ModernMailPublicFolders.ps1 (for Exchange 2013/2016/2019) to sync the mail-enabled public folders first.
```

File: Exchange/ExchangeOnline/collaboration-exo/public-folders/assign-permissions-mail-enabled-pfs.md (hunk 5 above)
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/collaboration-exo/public-folders/assign-permissions-mail-enabled-pfs.md:67]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: Exchange/ExchangeOnline/collaboration-exo/public-folders/assign-permissions-mail-enabled-pfs.md [hunk 8]
```diff
@@ -55,3 +55,22 @@ For detailed syntax and parameter information, see the following articles:
...
+
+2) Use following command in EXO PowerShell to assign SendAs permission:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/collaboration-exo/public-folders/assign-permissions-mail-enabled-pfs.md:74]:
```suggestion
2) Use the following command in EXO PowerShell to assign SendAs permission:
```

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**